app/src/main/java/org/bostwickenator/ftpuploader/FtpClient.java

```diff
@@ -9,43 +9,58 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 
+import static java.lang.Integer.parseInt;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_PASSIVE;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_PASSWORD;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_PORT;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_SERVER;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_TARGET_DIR;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_USERNAME;
+import static org.bostwickenator.ftpuploader.SettingsStore.settingsStore;
+
 class FtpClient {
 
     private final FTPClient ftpClient = new FTPClient();
     private final static String EMPTY_STRING = "";
-    SettingsStore settingsStore = SettingsStore.getSettingsStore();
+    private SettingsStore settingsStore = settingsStore();
```

Comment:
See other comment re name refactoring

Reply:
True. I thought about removing this line of code and using settingsStore() everywhere else instead. But thought this to be too offensive. :-) But would like it better though...